app/src/components/tables/ValidatorsTable/ValidatorsTable.test.tsx

```diff
@@ -52,38 +69,38 @@ describe('ValidatorsTable', () => {
   });
 
   it('should render a Rank', () => {
-    const rank = screen.getByTestId('rank');
-    expect(rank).toHaveTextContent(
+    const rank = screen.getAllByTestId('rank');
+    expect(rank[0]).toHaveTextContent(
```

Comment:
Why are we accessing the zeroth index for all the accessors?

Reply:
It guarantees that a single element is accessed if the number of rows are more than one.